ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -385,7 +385,7 @@ instance (ByronGiven, Typeable cfg, ConfigContainsGenesis cfg)
       lvUB = SlotNo $ unSlotNo currentSlot + (2 * paramK)
       lvLB
         | 2 * paramK > unSlotNo currentSlot
-        = genesisSlotNo
+        = SlotNo 0
```

Comment:
@nc6 Is this correct?

Reply:
Since the lower bound is non-strict, it should be fine.